src/main/java/salesforce/ui/pages/opportunity/NewOpportunityPage.java

```diff
@@ -0,0 +1,132 @@
+package salesforce.ui.pages.opportunity;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import salesforce.ui.pages.BasePage;
+
+public class NewOpportunityPage extends BasePage {
+
+    @FindBy(xpath = "//input[@name='IsPrivate']")
+    private WebElement privateCheckBox;
+
+    @FindBy(xpath = "//input[@name='CloseDate']")
+    private WebElement closeDate;
+
+    @FindBy(xpath = "//textarea[@class='slds-textarea']")
+    private WebElement textAreaInput;
+
+    @FindBy(xpath = "//button[@name='SaveEdit']")
+    private WebElement saveBtn;
+
+    @FindBy(name = "SaveAndNew")
+    private WebElement saveAndNewBtn;
+
+    @FindBy(name = "CancelEdit")
+    private WebElement cancelBtn;
+
+    private static final String XPATH_INPUT = "//input[@name='%s']";
+    private static final String XPATH_DROPDOWN_ELEMENT = "//label[text()='%s']/..//input";
+    private static final String XPATH_DROPDOWN_OPTION = "//lightning-base-combobox-item[@data-value='%s']";
+    //private static final String XPATH_SEARCH = "//input[contains(@placeholder,'%s')]";
```

Comment:
If these won't be used it should be deleted

Reply:
ok, it's done, thanks :)